package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "react-gift-react-foundation",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/babel-plugin": "^11.11.0",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.101",
+    "@types/react": "^18.3.3",
+    "@types/react-dom": "^18.3.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "craco start",
+    "build": "craco build",
+    "test": "craco test",
+    "eject": "craco eject",
+    "format": "prettier --write --cache src/**/*",
+    "lint": "eslint --cache .",
```

```suggestion
    "lint": "eslint --cache src",
```

lint 실행해보니 에러가 많이 발생하네요. 아마 node_modules에 포함된 코드도 lint 대상으로 잡혀서 그런거 같은데 한번 살펴보시겠어요? 위에 처럼 수정하면 아마 될 거 같은데 체크 부탁드리겠습니다.